applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_2D2N.cpp

```diff
@@ -279,12 +271,8 @@ namespace Kratos
 		ProcessInfo& rCurrentProcessInfo)
 		{
 			KRATOS_TRY;
-			if(this->mIsLinearElement) this->CalculateRightHandSideLinear(rRightHandSideVector,this->K_master);
-			else
-			{
-				rRightHandSideVector = ZeroVector(msElementSize);
-				rRightHandSideVector -= this->F_int_global;
-			}
+			rRightHandSideVector = ZeroVector(msElementSize);

```

Comment:
here it will silently resize if the size does not match, but the price for it is to create a temporary. 
If you are sure the size is already correct, then do

        rRightHandSideVector.clear()

or
        
     noalias(rRightHandSideVector) = ZeroVector(msElementSize);

Reply:
my reaction forces are 0 if I do this